testdata/configtest.json

```diff
@@ -845,103 +845,94 @@
SDT,SID_USUSDT,VPA_DUSDT,GGM_USDT,LOO_KSUSDT,MBS_USDT,DAI_USDT,BUS_DUSDT,ACA_USDT,MVU_SDT,MIX_USDT,LTC_USDC,MAN_ABTC,MAT_ICBTC,LTC_BTC,DOT_BTC,SAN_DBTC,MAN_AUSDC,MAT_ICUSDC,SAN_DUSDC,DOT_USDC,LUN_CUSDC,RSS_3USDT,SYN_RUSDT,TAP_USDT,ERT_HAUSDT,GMX_USDT,TUS_DT,ACH_USDT,JST_USDT,SUN_USDT,BTT_USDT,TRX_USDT,NFT_USDT,POK_TUSDT,SCR_TUSDT,PST_AKEUSDT,SON_USDT,HER_OUSDT,DOM_EUSDT,UST_CUSDT,BNB_USDT,NEA_RUSDT,PAX_GUSDT,SDU_SDT,APE_USDT,BTC_3SUSDT,BTC_3LUSDT,FID_AUSDT,MIN_AUSDT,SCU_SDT,RAC_AUSDT,CAP_SUSDT,STG_USDT,GLM_RUSDT,MOV_RUSDT,ZAM_USDT,ETH_DAI,BTC_DAI,WBT_CUSDT,XAV_AUSDT,MEL_OSUSDT,GMT_USDT,GST_USDT,CEL_OUSDT,SFU_NDUSDT,ELT_USDT,LGX_USDT,APE_XUSDT,CTC_USDT,COT_USDT,KMO_NUSDT,PLY_USDT,XWG_USDT,FIT_FIUSDT,STR_MUSDT,GAL_USDT,ETH_3SUSDT,ETH_3LUSDT,KOK_USDT,FAM_EUSDT,XRP_3SUSDT,XRP_3LUSDT,USD_DUSDT,OPU_SDT,LUN_AUSDT,DFI_USDT,MOV_EZUSDT,THN_USDT,DOT_3SUSDT,DOT_3LUSDT,VIN_UUSDT,BEL_USDT,FOR_TUSDT,AVA_X2SUSDT,AVA_X2LUSDT,ADA_2SUSDT,ADA_2LUSDT,WLK_NUSDT,KON_USDT,LTC_2SUSDT,LTC_2LUSDT,SAN_D2SUSDT,SAN_D2LUSDT,OBX_USDT,SEO_RUSDT,MNZ_USDT,CUL_TUSDT,DOG_EUSDC,EOS_USDC,CUS_DUSDT,SLG_USDT,CMP_USDT,KUN_CIUSDT,GST_SUSDT,XET_AUSDT,AZY_USDT,MMC_USDT,FLO_KIUSDT,BAB_YDOGEUSDT,STA_TUSDT,SAI_TAMAUSDT,MAT_IC2SUSDT,MAT_IC2LUSDT,ETC_2SUSDT,ETC_2LUSDT,DIC_EUSDT,WAX_PUSDT,ARU_SDT,KDA_USDT,ROS_EUSDT,SLG_USDC,APE_2SUSDT,APE_2LUSDT,GMT_2SUSDT,GMT_2LUSDT,DEF_YUSDT,PSG_USDT,BAR_USDT,JUV_USDT,ACM_USDT,INT_ERUSDT,AFC_USDT,CIT_YUSDT,LIN_K2LUSDT,LIN_K2SUSDT,FTM_2LUSDT,FTM_2SUSDT,SOL_OUSDT,WBT_CBTC,AVA_XUSDC,ADA_USDC,OPU_SDC,DOG_E2SUSDT,DOG_E2LUSDT,ATO_M2SUSDT,ATO_M2LUSDT,APE_XUSDC,TRX_USDC,ICP_USDC,LIN_KUSDC,GMT_USDC,CHZ_USDC,SHI_BUSDC,LDO_USDC,APE_USDC,FIL_USDC,CHR_PUSDT,EOS_2SUSDT,EOS_2LUSDT,WWY_USDT,LIN_GUSDT,SWE_ATUSDT,DLC_USDT,OKG_USDT,ETH_WUSDT,INJ_USDT,MPL_XUSDT,MIB_RUSDT,COU_SDT,AGL_AUSDT,RON_DUSDT,QMA_LLUSDT,PUM_LXUSDT,GCA_KEUSDT,APT_USDT,APT_USDC,USD_TEUR,MTK_USDT,MCR_TUSDT,MAS_KUSDT,ECO_XUSDT,HFT_USDC,HFT_USDT,KCA_LUSDT,PEO_PLEUSDT,TWT_USDT,ORT_USDT,HOO_KUSDT,PRI_MALUSDT,MCT_USDT,OAS_USDT,MAG_ICUSDT,MEE_USDT,TON_USDT,BON_KUSDT,FLR_USDT,TIM_EUSDT,3PU_SDT,RPL_USDT,SSV_USDT,FXS_USDT,COR_EUSDT,RDN_TUSDT,BLU_RUSDT,LIS_USDT,AGI_XUSDT,MDA_OUSDT,ACS_USDT,HVH_USDT,GNS_USDT,DPX_USDT,PIP_USDT,PRI_MEUSDT,EVE_RUSDT,VRA_USDT,GPT_USDT,FBU_SDT,DZO_OUSDT,IDU_SDT,ARB_USDC,ARB_USDT,XCA_DUSDT,MBX_USDT,AXL_USDT,CGP_TUSDT,PLA_YUSDT,AGI_USDT,RLT_MUSDT,SUI_USDT,SUI_USDC,TAM_AUSDT,MVL_USDT,PEP_EUSDT,LAD_YSUSDT,LMW_RUSDT,BOB_USDT,TOM_IUSDT,KAR_ATEUSDT,SUI_AUSDT,TUR_BOSUSDT,FMB_USDT,CAP_OUSDT,TEN_ETUSDT,VEL_OUSDT,ELD_AUSDT,CAN_DYUSDT,FON_USDT,OMN_USDT,TOM_SUSDT,MTC_USDT,VEL_AUSDT,USD_TBRZ,BTC_BRZ,PEN_DLEUSDT,EGO_USDT,PEP_E2USDT,NYM_USDT,MNT_USDT,MNT_USDC,MNT_BTC,GSW_IFTUSDT,SAL_DUSDT,ARK_MUSDT,NEO_NUSDT,WLD_USDC,WLD_USDT,PLA_NETUSDT,DSR_UNUSDT,SPA_RTAUSDT,TAV_AUSDT,SEI_LORUSDT,SEI_USDT,CYB_ERUSDT,ORD_IUSDT,KAV_AUSDT,VVU_SDT,SAI_LUSDT,PYU_SDUSDT,SOL_EUR,USD_CEUR,ADA_EUR,DOG_EEUR,LTC_EUR,XRP_EUR,ETH_EUR,BTC_EUR,VEX_TUSDT,CTT_USDT,NEX_TUSDT,KAS_USDT,NES_SUSDT,CAT_USDT,FET_USDT,LEV_ERUSDT,VEG_AUSDT,ZTX_USDT"
```

Comment:
These pairs need to be updated as any base currency with a len > 3 becomes invalid. Examples: DOG_EUSDT, PEO_PLEUSDT, PLA_YUSDT etc